File: public/code/Rj3JhGmyfTwN4mRq3FYz/4_check_even.cpp
#include<iostream>
using namespace std;

//++ Function to check the Number is Even or Odd
void check_even(int num){
    if( num == 0 ){
        cout<<"The Given Number is Zero"<<endl;
    }else if( num % 2 == 0){
        cout<<num<<" is Even"<<endl;
    }else{
        cout<<num<<" is Odd"<<endl;
    }
}

int main(){
    int num;
    cout<<"Enter the Number:";
    cin>>num;
    check_even(num);
    return 0;
}

File: public/code/Rj3JhGmyfTwN4mRq3FYz/4_check_even.py
# Function to Check the number is Even or Odd
def check_even(num):
    if num == 0:
        print("The Given Number is zero")
    elif num % 2 == 0:
        print(f"{num} is Even")
    else:
        print(f"{num} is Odd")
    
# Get user input an number
num = int(input("Enter the Number:"))
check_even(num)